### ISSUE TITLE
The "Cyborg" perk is missing

### ISSUE THREAD

**Issue by ThatAlphaPigeon** [Crispsy/Unofficial-Cyberpunk-2077-Build-Creator#1]
![image](https://github.com/user-attachments/assets/98292cae-3c88-4582-b3e2-c37320a6508d)
![image](https://github.com/user-attachments/assets/d41c079b-fb96-41d3-9685-57bc8358033d)

**Comment by Crispsy** [comment]
Yeah... That's a small mistake I made with early versions that caused it to not show up now.

Luckily, it doesn't affect any stats in the creator. But it's one of several things wrong with it. I don't plan on making a different version, I'd rather work on making it a mod instead... I learned a lot from creating this one but there are so many things wrong with it I'd basically be rewriting it.

However, if you want it fixed for yourself, you can edit the main.py file. In the techScreenPerk func, change the value after "cyberwareLTC" to 6 (or higher), running the main.py file afterwards should fix it

Apologies for the mistake! Thanks for commenting!
